js/src/edit-free-campaign/index.js

```diff
@@ -134,7 +134,12 @@ const EditFreeCampaign = () => {
 		savedShippingRates
 	);
 
-	const didTimesChanged = ! isEqual( shippingTimes, savedShippingTimes );
+	// Dont take in consideration the order of the rules when comparing the Shipping times.
```

Comment:
💅

I would say "Dont take in consideration the order of the shipping times" as I think they are not rules. 

Or something similar to:

https://github.com/woocommerce/google-listings-and-ads/blob/99bfa1819e680a35aee18e1da0a9e2fa998db41f/js/src/edit-free-campaign/index.js#L123